### DISCUSSION TITLE
Welcome to flamegapps Discussions!

### DISCUSSION THREAD

**Discussion by ayandebnath** [flamegapps/flamegapps#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ravi0900** [comment]
Hi I'm ravi, It just a feedback for you pro devs, I love to use flame gapps with custom rom and I must say when flame gapps basic is around 90mb size, custom rom run so smoothly in my 3gb ram device 😊, but when around 100+ mb, it lag a bit, I recently flashed android 13 with that canary 94mb gapps build and ROM run like super smooth but overall I loved using this gapps and will keep using this, thank you for making this nice gapps packs ❤️.
